<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*全局设置*/
* {	font-family: &quot;微软雅黑&quot;; font-size: 15px;}

/*最上边的Logo图标*/
QLabel#labelLogo {
	background: #569082;
	outline:none;
	color:#333;
}
/*再来一个Logo动画*/
QLabel#Logo{
	background: #009082;
	outline:none;
	color:#333;
}

/*注册*/
QLabel#labelRegist {
	color: rgb(0, 170, 127);
}
QLabel#labelRegist:hover
{
    color: rgb(255, 0, 0);
	font: 75 20pt &quot;微软雅黑&quot;;

}

/*忘记密码*/
QLabel#labelForget {
	color: rgb(0, 170, 127);

}
QLabel#labelForget:hover
{
    color: rgb(255, 0, 0);
	font: 75 20pt &quot;微软雅黑&quot;;

}


/*最小化按钮*/
QPushButton#btnMin {
	background: transparent;
	border: none;
	color: #ffffff;
	font-size: 18px;
}
QPushButton#btnMin:hover {

	background: #569000;
}


/*关闭按钮*/
QPushButton#btnClose {
	background: transparent;
	border: none;
	color: #ffffff;
	font-size: 18px;
}
QPushButton#btnClose:hover {
	background: red;
}


/*注册按钮*/
QPushButton#btnLogin {
	border: 1px solid #c9c9c9;
	background: #569082;
	color:#ffffff;
	font-family: &quot;微软雅黑&quot;;
	font-size: 15px;
}
QPushButton#btnLogin:hover {
	border: 1px solid #c9c9c9;
	background: #ff9032;
	color:#ffffff;
	font-family: &quot;微软雅黑&quot;;
	font-size: 15px;
   
}
QPushButton#btnLogin:pressed {
	border:9px solid #c9c9c9;
	background: #569082;
	color:#ffffff;
	font-family: &quot;微软雅黑&quot;;
	font-size: 20px;
   
}


/*记住密码复选框*/
QCheckBox::indicator 
{
	width: 15px;
 	height: 15px;
	margin-right:5px;
}
QCheckBox#cbkAuto {
	font-family: &quot;微软雅黑&quot;;
	font-size: 15px;
	color: rgb(46, 44, 45);
}
QCheckBox#cbkAuto:hover {
	font-family: &quot;微软雅黑&quot;;
	font-size: 15px;
	
	color: rgb(0, 206, 0);
}
QCheckBox#cbkAuto:checked {
	font-family: &quot;微软雅黑&quot;;
	font-size: 15px;
	
	color: rgb(0, 170, 127);
}
/*输入框全局设置*/
QLineEdit {
	border: none;    
	font: 75 15pt &quot;微软雅黑&quot;;
	background: transparent;
}
/*输入用户名父控件*/
QWidget#widgetUser {
	border: 1px solid #c9c9c9;

}

/*输入密码父控件*/
QWidget#widgetPassword {
	border: 1px solid #c9c9c9;
}





</string>
  </property>
  <widget class="QLabel" name="labelLogo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#labelLogo {
	background: #569082;
	outline:none;
	color:#333;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>332</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMin">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRegist">
   <property name="geometry">
    <rect>
     <x>405</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>注册帐号</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelForget">
   <property name="geometry">
    <rect>
     <x>406</x>
     <y>249</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>忘记密码</string>
   </property>
  </widget>
  <widget class="QWidget" name="widgetPassword" native="true">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>240</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEditPassword">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>0</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>5</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">label{
image: url(:/resource/image/login/resource/image/login/passwd.png);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/resource/image/login/resource/image/login/passwd.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetUser" native="true">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>180</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>5</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">label1{

url(:/resource/image/login/resource/image/login/person.png)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/resource/image/login/resource/image/login/person.png</pixmap>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUser">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>0</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>15</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMask">
     <string extracomment="请输入数字"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QMenuButton" name="btnStatus">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>-2</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>2</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    border:none;
}
QPushButton:hover{
    border:none;
}
QPushButton:pressed {

    border:none;
} </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/resource/image/login/resource/image/login/svg/status-online-1.svg</normaloff>:/resource/image/login/resource/image/login/svg/status-online-1.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="cbkAuto">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>290</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="Logo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>48</y>
     <width>530</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPhoto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(255, 120, 31);
border-image:  url(:/resource/image/login/resource/image/login/svg/avatar-border.svg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/resource/image/login/resource/image/login/avatar.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbkAutoLogin">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>290</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*记住密码复选框*/
QCheckBox::indicator 
{
	width: 15px;
 	height: 15px;
	margin-right:5px;
}
QCheckBox{
	font-family: &quot;微软雅黑&quot;;
	font-size: 15px;
	color: rgb(46, 44, 45);
}
QCheckBox:hover {
	font-family: &quot;微软雅黑&quot;;
	font-size: 15px;
	
	color: rgb(0, 206, 0);
}
QCheckBox:checked {
	font-family: &quot;微软雅黑&quot;;
	font-size: 15px;
	
	color: rgb(0, 170, 127);
}</string>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
  </widget>
  <zorder>labelLogo</zorder>
  <zorder>btnClose</zorder>
  <zorder>btnLogin</zorder>
  <zorder>btnMin</zorder>
  <zorder>labelRegist</zorder>
  <zorder>labelForget</zorder>
  <zorder>widgetPassword</zorder>
  <zorder>widgetUser</zorder>
  <zorder>cbkAuto</zorder>
  <zorder>Logo</zorder>
  <zorder>cbkAutoLogin</zorder>
  <zorder>labelPhoto</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMenuButton</class>
   <extends>QPushButton</extends>
   <header location="global">qmenubutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
